pem/lib/pem/options.rb

```diff
@@ -18,9 +18,16 @@ def self.available_options
                                      description: "Generate a p12 file additionally to a PEM file",
                                      is_string: false,
                                      default_value: true),
+        FastlaneCore::ConfigItem.new(key: :current_active_days_limit,
+                                     env_name: "PEM_CURRENT_ACTIVE_DAYS_LIMIT",
+                                     description: "If current one is active for no more than this number of days, generate a new one. Default value is 30",
```

Comment:
Could we change this description to say "If the current certificate is active for less than this number of days, generate a new one. Default value is 30 days."